src/photod/column_map/catalog.py

```diff
@@ -0,0 +1,9 @@
+"""
+Column map for catalog data.
+"""
+
+from photod.column_map.base import mapper_from_glossary
+from pathlib import Path
+
+glossary_path = Path(__file__).parent / "glossary.yaml"
```

Comment:
I'm still confused about how this would be used in practice. The intent behind the ColumnMapper is that each survey will have different names for each field, and even different scientists using the same survey may have renamed their columns to fit their own language. We want users to specify which columns in their special datasets should be used in the distance calculations.

If I'm running this pipeline for SDSS, then will run for LSST the next day to compare results, how would I do that? Would I have to change this file, and re-build from source?

Reply:
That's a great point.  I got hung up on trying to show how it could be defaulted for a project, but it's buried too low.  It certainly can be configured at the point of use and I will change it to demonstrate that.